### PR TITLE
Added support internal classes to JavaParserTypeSolver

### PR DESCRIPTION
**Pull request by mlangkabel**

### CODE REVIEW

File: java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparser/Navigator.java [hunk 1]
```diff
@@ -23,8 +23,34 @@ private Navigator() {
         if (cu.getTypes() == null) {
             return Optional.empty();
         }
-        return cu.getTypes().stream().filter((t) -> t.getName().equals(name)).findFirst();
+        
+        String[] nameParts = name.split("\\.", 2);
```
Comment (ftomassetti) [inline java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparser/Navigator.java]: This name splitting seems to be repeated in several points, maybe we want to create an helper method for that.

File: java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparser/Navigator.java [hunk 2]
```diff
@@ -23,8 +23,34 @@ private Navigator() {
...
+        final String typeName = nameParts[0];
+        
+        Optional<TypeDeclaration> type = cu.getTypes().stream().filter((t) -> t.getName().equals(typeName)).findFirst();
+        
+        if (nameParts.length > 1 && type.isPresent()) {
+            return findType(type.get(), nameParts[1]);
```
Comment (ftomassetti) [inline java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparser/Navigator.java]: We could assign nameParts[1] to a variable so the name would make more obvious what it is

File: java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparser/Navigator.java (hunk 1 above)
Comment (mlangkabel) [inline java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparser/Navigator.java]: Sure! I can look into that tomorrow.

**Comment by ftomassetti** [comment]
Great, thanks!

**Pull request merged**